database/src/pool/postgres.rs

```diff
@@ -309,6 +309,28 @@ static MIGRATIONS: &[&str] = &[
     // Prevent multiple try commits without a `sha` and the same `pr` number
     // being added to the table
     r#"CREATE UNIQUE INDEX benchmark_request_pr_commit_type_idx ON benchmark_request (pr, commit_type) WHERE status != 'completed';"#,
+    r#"
+    CREATE TABLE IF NOT EXISTS job_queue (
+        id            SERIAL PRIMARY KEY,
+        request_id    INTEGER NOT NULL,
+        target        TEXT NOT NULL,
+        backend       TEXT NOT NULL,
+        benchmark_set INTEGER NOT NULL,
+        collector_id  TEXT NOT NULL,
```

Comment:
We finally got to a place where we need collectors, let's merge https://github.com/rust-lang/rustc-perf/pull/2157 first then, so that we can use a proper foreign key here.